CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 **Features**:
 
+- Session metrics extraction: Always extract distinct_id. ([#1275](https://github.com/getsentry/relay/pull/1275))
```

Comment:
Would classify this under bug fixes and describe what the user-visible effect of this is, or rather what the bug was.